public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.test.tsx

```diff
@@ -40,18 +40,21 @@ describe('MetricStatEditor', () => {
     config.featureToggles.cloudWatchCrossAccountQuerying = originalFeatureToggleValue;
   });
   describe('statistics field', () => {
-    test.each(['Average', 'p23.23', 'p34', '$statistic'])('should accept valid values', async (statistic) => {
-      const onChange = jest.fn();
-
-      render(<MetricStatEditor {...props} onChange={onChange} />);
-
-      const statisticElement = await screen.findByLabelText('Statistic');
-      expect(statisticElement).toBeInTheDocument();
-
-      await userEvent.type(statisticElement, statistic);
-      fireEvent.keyDown(statisticElement, { keyCode: 13 });
-      expect(onChange).toHaveBeenCalledWith({ ...props.metricStat, statistic });
-    });
+    test.each([['Average', 'p23.23', 'p34', 'TM(2%:98%)', 'WM(10%:90%)', 'TS(80%:)', '$statistic']])(
```

Comment:
```suggestion
    test.each(['Average', 'p23.23', 'p34', 'TM(2%:98%)', 'WM(10%:90%)', 'TS(80%:)', '$statistic'])(
```

This suggestion lets each element in the array be run in the test. It seemed to have been originally written incorrectly as an array of array. It just needs to be one array.

Reply:
Yeah fixed this.

Reply:
@sladyn98 don't see your latest changes yet